### PR TITLE
Prevent people from tagging @everyone or @here in the Discord bot

### PR DESCRIPTION
**Pull request by connor4312**
removes the @ sign:

### CODE REVIEW

**Comment by MattJustMatt** [comment]
@Doridian let's get this deployed first thing tomorrow

File: src/replaceWords.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * A map of words that should be replaced in Beam chat messages.
+ */
+const wordReplacements = new Map<RegExp, string>([
+    [/@everyone/g, 'everyone'],
```
Comment (SimonSchick) [inline src/replaceWords.ts]: You could make that a single regex :P

File: tsconfig.json [hunk 2]
```diff
@@ -2,8 +2,12 @@
 	"compilerOptions": {
 		"declaration": true,
 		"module": "commonjs",
-		"target": "es5",
+		"target": "es6",
 		"noImplicitAny": true,
-		"outDir": "lib"
+		"outDir": "lib",
```
Comment (SimonSchick) [inline tsconfig.json]: Weird indenting

File: src/replaceWords.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    [/@here/g, 'here'],
+]);
+
+/**
+ *
+ */
+export function replace(source: string): string {
+    wordReplacements.forEach((replacement, pattern) => {
```
Comment (SimonSchick) [inline src/replaceWords.ts]: Could use reduce instead.

File: src/replaceWords.ts (hunk 1 above)
Comment (connor4312) [inline src/replaceWords.ts]: I could, but I wanted to keep this clean and expandable if we ever want to add more such words or tags in the future

**Pull request merged**